### PR TITLE
Compile on Ubuntu 18.04

### PR DESCRIPTION
**Pull request by AltraMayor**
The previous LTS version of Ubuntu, namely 18.04, is still in use in production. While we cannot guarantee to always support older versions of Ubuntu or any other distribution, this pull request is simple and small enough to warranty it.

### CODE REVIEW

File: lib/net.c [hunk 1]
```diff
@@ -1775,6 +1775,27 @@ start_network_stage2(void *arg)
 	return ret;
 }
 
+/* These definitions are meant to support older versions of libcap. */
+#ifndef CAP_AMBIENT_SUPPORTED
+
+#define CAP_AMBIENT_SUPPORTED() (0)
+
+static inline int
+cap_get_ambient(cap_value_t x)
```
Comment (andrenth) [inline lib/net.c:1784]: This causes an unused argument warning, leading to a compilation error due to -Wfatal-errors:
```
gatekeeper/lib/net.c: In function ‘cap_get_ambient’:
gatekeeper/lib/net.c:1784:29: error: unused parameter ‘x’ [-Werror=unused-parameter]
 cap_get_ambient(cap_value_t x)
```

**Comment by andrenth** [comment]
The `cap_setgroups(3)` function is also not available on Ubuntu 18.04's version of libcap (2.25).

**Comment by AltraMayor** [comment]
There's no easy fix for `cap_setgroups(3)`, so we are going to abandon Ubuntu 18.04 and move to the current LTS version, namely 20.04.

**Comment by AltraMayor** [comment]
Pull request #478 supersedes this pull request.

**Pull request closed without merging**